sdk/python/feast/expediagroup/vectordb/elasticsearch_online_store.py

```diff
@@ -96,8 +97,22 @@ def online_write_batch(
         ],
         progress: Optional[Callable[[int], Any]],
     ) -> None:
-        with ElasticsearchConnectionManager(config):
-            pass
+        with ElasticsearchConnectionManager(config) as es:
+            resp = es.indices.exists(index=table.name)
+            if not resp.body:
+                self._create_index(es, table)
+            bulk_documents = []
+            for entity_key, values, timestamp, created_ts in data:
+                id_val = self._get_value_from_value_proto(entity_key.entity_values[0])
+                document = {entity_key.join_keys[0]: id_val}
+                for feature_name, val in values.items():
+                    document[feature_name] = self._get_value_from_value_proto(val)
+                bulk_documents.append(
+                    {"_index": table.name, "_id": id_val, "doc": document}
+                )
+
+            helpers.bulk(client=es, actions=bulk_documents)
+            es.indices.refresh(index=table.name)
```

Comment:
this is a synchronous call. We may want to let ElasticSearch automatically refresh: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-refresh.html#refresh-api-desc. What do you think?